meteor/server/api/showStyles.ts

```diff
@@ -94,6 +95,26 @@ export async function insertShowStyleVariantInner(
 		_rundownVersionHash: '',
 	})
 }
+export async function insertShowStyleVariantWithBlueprint(
```

Comment:
This is nearly identical with `insertShowStyleVariant`. Duplicated code is bad and this should be a red flag for you.
Either extract the common code into a separate method that both this and `insertShowStyleVariant` can use or add the optional parameter of the blueprintConfig to `insertShowStyleVariant`.

Reply:
I very much agree, i want to change this, but do you know whether it is important for the `id` and `rundownVersionHash` to be added when importing variants? If they do not matter it is much simpler to make them use the same implementation by making the `blueprintConfig` optional.